README.md

```diff
@@ -2,6 +2,40 @@
 
 Enables validation for inputs in conjunction with hyper-validator modules.
 
+## Usage
+
+This is the base for all the validator modules. It collects the different validator modules and runs the
+validattions at the right time. To install it simply do `npm install hyper-validator-base --save`.
+This will add the validator base in your *node_modules* folder and insert it as a dependecy in *package.json*.
+
+Remember to load the file in you view, it's as simple as adding:
+
+```
+<script type="text/javascript" src="node_modules/hyper-validator-base/src/hyper-validator-base.js"></script>
+```
+
+If you are using Gulp or Grunt this process might be automated.
+
+To initialize it you need to inject `hyper-input-validator` as a dependecy in your angular module. Like this:
+
+```
+angular.module("hyperCoolApp", ['hyper-input-validator']);
+```
+
+And now you should be all good.
+
+## Modules
+
+Now you are ready to install the individual input validators. At the moment there is 1 you can add:
```

Comment:
Could we change this into 'one'?

> With very little deviation, most grammatical texts rule that the numbers zero to nine inclusive should be "written out" – meaning instead of "1" and "2", one would write "one" and "two"